buttercup/strings/en_US.yaml

```diff
@@ -152,6 +152,22 @@ heatmap:
   plot_ylabel: Weekday
   response_message: |
     Here is the heatmap for {user} {time_str}! ({duration})
+activity:
+  getting_activity: |
+    Generating the yearly activity map for {user}...
+  days:
+    - Mon
+    - Tue
+    - Wed
+    - Thu
+    - Fri
+    - Sat
+    - Sun
+  plot_title: Yearly activity map for {user} {time}
+  plot_xlabel: Time ({timezone})
+  plot_ylabel: Weekday
+  response_message: |
+    Here is the yearly activity map for {user} {time}! ({duration})
```

Comment:
wouldn't this read a bit funny on the user end?

Reply:
Here's how the message looks like on the user side:
![Here is the yearly activity map for everyone from 1 year ago until now! (2113 ms)](https://user-images.githubusercontent.com/13908946/184928974-b95f93d4-3098-4798-a45d-344437d4465b.png)

Please let me know if you have suggestions on how to improve that it! :)